gui/db_tool/query_page/query_page.cpp

```
@@ -1,40 +1,64 @@
+QueryPage::QueryPage(QWidget *parent) : QWidget(parent), model(new QSqlQueryModel(this))
 {
-    ui->setupUi(this);
+    buildUi();
 
-    // DBの初期化
     db = QSqlDatabase::database();
 
     if (!db.open()) {
         QMessageBox::critical(this, "DB Error", "データベースに接続できませんでした");
         return;
     }
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)